.github/workflows/build.yml

```diff
@@ -0,0 +1,26 @@
+name: Build
+
+on:
+  push:
+    branches-ignore:
+      - release/* # already building package when publishing
+
+jobs:
+  build:
+    name: Build package (test)
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout Git repository
+        uses: actions/checkout@v2.3.4
+
+      - name: Setup Node
+        uses: actions/setup-node@v2.1.5
+        with:
+          node-version: 14
```

Comment:
Will this workflow be updated to get the version from `.nvmrc` file like the other workflows as a separate PR after it is merged?

Reply:
Yep, rebasing now!